build-monitor-acceptance/build-monitor-acceptance-base/src/test/java/features/ShouldDescribeEachProject.java

```diff
@@ -47,7 +47,7 @@ public void displaying_a_custom_build_description() throws Exception {
                 Navigate.to(jenkins.url()),
                 HaveAProjectCreated.called("Example Github Project").andConfiguredTo(
                         ExecuteAShellScript.that(outputsGithubProjectLog()),
-                        SetBuildDescription.to("Revision: \\1").basedOnLogLineMatching("Checking out Revision ([^\\s]{6})")
+                        SetBuildDescription.to("Revision: \\1").basedOnLogLineMatching("Checking out Revision ([^\\w]{6})")
```

Comment:
@jan-molak I'm very unsure about this fix, but suddenly I just couldn't get this test to work for any version of Jenkins while it did work before.

And \s is whitespace, right? So it looks like it shouldn't work, but obviously it did before 😄

Reply:
oh yeah of course. hmmm this is so weird then. because it won't work for me with ^\\s, but it did before, and it WILL work with ^\\w but it shouldnt.